### PR TITLE
prep for shiny future

### LINKED ISSUE #116: prep for shiny future
We are going to start accepting shiny future stories! We need to do some prep work:

* [ ] update the [main "How to Vision" page](https://github.com/rust-lang/wg-async-foundations/blob/master/src/vision/how_to_vision.md) to mark that "shiny future pages" are being accepted
* [ ] update the [main "How to Shiny Future" page](https://github.com/rust-lang/wg-async-foundations/blob/master/src/vision/how_to_vision/shiny_future.md) to mark that "shiny future pages" are being accepted
* [ ] update the [shiny future](https://github.com/rust-lang/wg-async-foundations/blob/master/src/vision/shiny_future.md) page to direct to [have a section like this](https://github.com/rust-lang/wg-async-foundations/blob/master/src/vision/status_quo.md#-under-construction-help-needed-)
* [ ] update the [shiny future template](https://github.com/rust-lang/wg-async-foundations/blob/master/src/vision/shiny_future/template.md#-not-time-for-this-yet-) to not say that we are not accepting such stories
* [ ] create a github issue template similar to the [one for status quo issues](https://github.com/rust-lang/wg-async-foundations/blob/master/.github/ISSUE_TEMPLATE/-status-quo--story-issue.md)

### PR DESCRIPTION
**Pull request by Frederik-Baetens**
Do the issue templates need to start with a - ? That makes working with them a bit annoying.

resolves #116

### CODE REVIEW

File: src/vision/how_to_vision/shiny_future.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # ❓ How to vision: "Shiny future" stories
 
-## 🛑 Not time for this yet 🛑
-
-We're not ready for this yet! See the [how to vision](../how_to_vision.md) page for details on the phasing.
+We want all Async Rust users and their hopes and dreams for what Async Rust should be in the future to be reflected in the async vision doc, so please help us by writing 'shiny future' stories about what you would like async Rust to look like! Remember, **shiny future visions can be personal and may differ greatly from person to person. Therefore it's impossible for these visions to be wrong.**
```
Comment (nikomatsakis) [inline src/vision/how_to_vision/shiny_future.md:3]:
```suggestion
We want all Async Rust users and their hopes and dreams for what Async Rust should be in the future to be reflected in the async vision doc, so please help us by writing 'shiny future' stories about what you would like async Rust to look like! **Remember: we are in a brainstorming period.** Please feel free to leave comments in an effort to help someone improve their PRs, but if you would prefer a different approach, you are better off writing your own story. (In fact, you should write your own story even if you like their approach but just have a few alternatives that are worth thinking over.)
```

Maybe like this?

**Review by nikomatsakis — CHANGES_REQUESTED** [review]
Thanks @Frederik-Baetens! This looks great. I am doing a quick look over the page to look for other things that should be updated:

* [This page](https://rust-lang.github.io/wg-async-foundations/vision/how_to_vision.html) says "Folks open "status quo" and (starting 2021-04-02) "shiny future" story PRs", we can probably remove the parenthetical.

That looks like it for now.

**Comment by nikomatsakis** [comment]
Thanks @Frederik-Baetens! Note to all, I'd like to hold off on merging this until we are ready.

Regarding the names of issue templates, I have no idea. I think I created it with the web gui and it did what it did. =)

File: src/vision/how_to_vision/shiny_future.md (hunk 1 above)
Comment (Frederik-Baetens) [reply src/vision/how_to_vision/shiny_future.md:3]: That seems better, I was also thinking that shiny future visions could be a little more "wrong" than status quo stories.

Comment (Frederik-Baetens) [reply src/vision/how_to_vision/shiny_future.md]: I also removed the parenthesis in the sentence you mentioned. Do you want parenthesis removed else where as well?

Comment (Frederik-Baetens) [reply src/vision/how_to_vision/shiny_future.md]: Oh, I misunderstood what you meant, actually removed the parenthetical now :sweat_smile:

**Review by nikomatsakis — APPROVED** [review]
Looks good, thanks! Let's hold off on merging till we're all ready though.

**Review by nikomatsakis — APPROVED** [review]

**Pull request merged**